src/v/config/configuration.cc

```diff
@@ -825,6 +825,13 @@ configuration::configuration()
        .visibility = visibility::user},
       {},
       validate_connection_rate)
+  , kafka_rpc_server_stream_recv_buf(
+      *this,
+      "kafka_rpc_server_stream_recv_buf",
+      "Userspace receive buffer max size in bytes",
+      {.example = "65536", .visibility = visibility::tunable},
+      std::nullopt,
+      {.min = 32_KiB, .align = 4_KiB})
```

Comment:
should we add max value here as well ?

Reply:
Yes, good call.  I've set it to 512kB as an arbitrary "huge but has a chance of getting allocated" value.  It should at least prevent people totally breaking their system by setting 100MB here.

I also tweaked the minimum value: my previous 32kB was arbitrary, but there is already a hard minimum inside seastar at 512 bytes, so I've changed it to that.